pkg/skuba/actions/auth/auth.go

```diff
@@ -203,33 +209,50 @@ func doAuth(authReq request) (*response, error) {
 	}
 	defer resp.Body.Close()
 
-	z := html.NewTokenizer(resp.Body)
-
-	var actionLink string
-
-Loop:
-	for {
-		tt := z.Next()
+	respDump, err := httputil.DumpResponse(resp, true)
+	if err != nil {
+		return nil, errors.Wrapf(err, "http dump response")
+	}
 
-		switch {
-		case tt == html.ErrorToken:
-			// End of the document, we're done
-			break Loop
-		case tt == html.StartTagToken || tt == html.SelfClosingTagToken:
-			t := z.Token()
+	loginURL := authReq.IssuerURL
+	if strings.Contains(string(respDump), singleConnectorMsg) {
```

Comment:
I thought of something after approving:
We don't seem to protect for a failed response here.  If this [Get](https://github.com/SUSE/skuba/blob/7567b42a6a3fee81cf56dcefe60c8d98bbf24fbb/pkg/skuba/actions/auth/auth.go#L206) fails, we land in this [else](https://github.com/SUSE/skuba/blob/7567b42a6a3fee81cf56dcefe60c8d98bbf24fbb/pkg/skuba/actions/auth/auth.go#L222) block meant for multiple connectors.

Reply:
Good point, let me think if there is any better way to check single connector or multiple connectors or others response.

Reply:
Add one more case if the response is not single connector or multiple connectors